web/core/components/cycles/active-cycle/productivity.tsx

```diff
@@ -19,22 +19,22 @@ export type ActiveCycleProductivityProps = {
 };
 
 const cycleBurnDownChartOptions = [
-  { value: "burndown", label: "Issues" },
+  { value: "issues", label: "Issues" },
   { value: "points", label: "Points" },
 ];
 
 export const ActiveCycleProductivity: FC<ActiveCycleProductivityProps> = observer((props) => {
   const { workspaceSlug, projectId, cycle } = props;
   // hooks
-  const { getPlotTypeByCycleId, setPlotType } = useCycle();
+  const { getEstimateTypeByCycleId, setEstimateType } = useCycle();
   const { currentActiveEstimateId, areEstimateEnabledByProjectId, estimateById } = useProjectEstimates();
 
   // derived values
-  const plotType: TCyclePlotType = (cycle && getPlotTypeByCycleId(cycle.id)) || "burndown";
+  const estimateType: TCycleEstimateType = (cycle && getEstimateTypeByCycleId(cycle.id)) || "issues";
```

Comment:
Will progressUnit be a better name?

Reply:
Its a judgement call, to me estimateType looks better